src/Commander-Activators-Shortcut/CmdShortcutActivation.class.st

```diff
@@ -59,6 +59,27 @@ CmdShortcutActivation class >> by: aKeyCombination for: anAnnotationUser [
 		keyCombination: aKeyCombination 
 ]
 
+{ #category : #'well known shortcuts' }
+CmdShortcutActivation class >> generateAccesors: anAnnotationUser [  
```

Comment:
This looks wrong.
Commander should know nothing about refactorings.

I understand that it looks similar to #renamingFor: and #removalFor: . But those words have general reusable meaning. You can reuse them for file operations for example

Accessors and pullUp/pullDown  are something tightly coupled to code changes. And they should be packaged into one of SystemCommands packages.

Reply:
From the other side they are used on single place in the system. I propose to inline and remove them.